packages/ozone/src/mod-service/lang.ts

```diff
@@ -0,0 +1,75 @@
+import { ModerationService } from '.'
+import { ModSubject } from './subject'
+import { ModerationSubjectStatusRow } from './types'
+
+export class ModerationLangService {
+  constructor(private moderationService: ModerationService) {}
+
+  async tagSubjectWithLang({
+    subject,
+    subjectStatus,
+    createdBy,
+  }: {
+    subject: ModSubject
+    createdBy: string
+    subjectStatus: ModerationSubjectStatusRow | null
+  }) {
+    if (
+      subjectStatus &&
+      !subjectStatus.tags?.find((tag) => tag.includes('lang:'))
+    ) {
+      try {
+        const recordLangs = await this.getRecordLang({
+          subject,
+        })
+        await this.moderationService.logEvent({
+          event: {
+            $type: 'com.atproto.admin.defs#modEventTag',
+            add: recordLangs
+              ? recordLangs.map((lang) => `lang:${lang}`)
+              : ['lang:unknown'],
+            remove: [],
+          },
+          subject,
+          createdBy,
+        })
+      } catch (err) {
+        console.error('Error getting record langs', err)
```

Comment:
We should use the logger here rather than `console.error()` so that we can consume the log in a standard format.